src/software/field_tests/field_test_fixture.py

```diff
@@ -0,0 +1,510 @@
+                            eventually_validation_sequence_set=[
+                                [ball_placement_validation_function]
+                            ],
+                            always_validation_sequence_set=[[]],
+                        )
+
+                        if not validation.contains_failure(
+                            eventually_validation_status
+                        ):
+                            logger.info("validations pass")
+                            break
+
+                    except queue.Empty as empty:
+                        logger.warning("failed to obtain world")
+
+                validation.check_validation(eventually_validation_status)
+
+            logger.info("moving robots to position")
+            # move robots to position
+            initial_position_tactics = [
+                AssignedTacticPlayControlParams(),
+                AssignedTacticPlayControlParams(),
+            ]
+
+            robot_positions_validation_functions = [ball_placement_validation_function]
+
+            for team_idx, team in enumerate(
+                [worldstate.blue_robots, worldstate.yellow_robots]
+            ):
+                for robot_id in team.keys():
+                    robotState = team[robot_id]
+                    move_tactic = MoveTactic()
+                    move_tactic.destination.CopyFrom(robotState.global_position)
+                    move_tactic.final_orientation.CopyFrom(
+                        robotState.global_orientation
+                        if robotState.HasField("global_orientation")
+                        else Angle(radians=0.0)
+                    )
+                    move_tactic.final_speed = 0.0
+                    move_tactic.dribbler_mode = DribblerMode.OFF
+                    move_tactic.ball_collision_type = BallCollisionType.AVOID
+                    move_tactic.auto_chip_or_kick.CopyFrom(
+                        AutoChipOrKick(autokick_speed_m_per_s=0.0)
+                    )
+                    move_tactic.max_allowed_speed_mode = (
+                        MaxAllowedSpeedMode.PHYSICAL_LIMIT
+                    )
+                    move_tactic.target_spin_rev_per_s = 0.0
+                    initial_position_tactics[team_idx].assigned_tactics[
+                        robot_id
+                    ].move.CopyFrom(move_tactic)
+
+                    # create validation
+                    expected_final_position = tbots.Point(
+                        robotState.global_position.x_meters,
+                        robotState.global_position.y_meters,
+                    )
+
+                    team_color = [Team.BLUE, Team.YELLOW][team_idx]
+                    validation_func = SpecificRobotEventuallyEntersRegion(
+                        robot_id=robot_id,
+                        team=team_color,
+                        regions=[tbots.Circle(expected_final_position, 0.1)],
+                    )
```

Comment:
I think one of our goals with our new pytest system was to make validation functions robot id agnostic, so I think RobotEventuallyEntersRegion should be reasonable enough in this situation to test the same thing?

In this case, I don't think coding the validation like this is a bad thing since the test fixture is setting the robot id itself for the validation but I'm wondering if RobotEventuallyEntersRegion is good enough